[draft] feat: new desktop layout for finality providers list

desamtralized opened the pull request.

desamtralized committed:
fix: OneKey Wallet Issues (#23)

* onekey fix attempt

* delete empty file

* fix(OneKeyWallet): Enable network switching and update API calls

- Uncomment and enable network switching logic based on networkEnv
- Update getAddress() to use getAccounts()[0] instead of getAddress()
- Update getPublicKeyHex() to use getPublicKey() instead of getPublicKeyHex()

* refactor: Update OneKeyWallet provider methods

- Simplify getWalletProviderName to return "OneKey"
- Implement getBTCTipHeight using getTipHeight from mempool API

* refactor: update OneKey wallet integration and utility functions

- Update OneKey wallet provider to use btcwallet instead of btc
- Refactor signPsbtTransaction utility and update its imports
- Adjust import paths for various utility functions
- Update OneKeyWallet class methods to align with new provider interface
- Remove unnecessary null checks and update method calls in OneKeyWallet
- Temporarily comment out network switching logic in OneKeyWallet
- Update getWalletProviderName, getAddress, getPublicKeyHex, and getBTCTipHeight methods in OneKeyWallet

* wip: handle non segwit/taproot addresses.

* fix: error provider wrapping all other providers.

desamtralized committed:
feat: new desktop layout for finality providers list

MattNguyen commented on src/app/stake/btc/page.tsx:
```diff
@@ -126,87 +121,146 @@ const StakeBTCPage = () => {
-              <div className="text-2xl ">
-                {satoshiToBtc(btcWalletBalanceSat)}
+        <div className="flex items-baseline mb-4">
+          <h1 className="text-[#332B29] text-2xl font-bold mr-[16px]">
```
we should using custom values like this. brand colors should be added to tailwinds config. we should have a sensible naming convention for them like surface, secondary, primary, etc.

MattNguyen commented on src/app/stake/btc/page.tsx:
```diff
@@ -126,87 +121,146 @@ const StakeBTCPage = () => {
+                <p className="text-[#332B29] text-[12px] font-extralight leading-normal">
+                  CAP
+                </p>
+                <p className="text-[#332B29] text-[20px] font-medium leading-normal">
```
text sizing should also be config based

desamtralized committed:
wip: staking dialog

desamtralized committed:
wip: showing staking dialog with new layout and proper data

desamtralized committed:
Refactor staking components and improve form handling

- Update reset prop type from boolean to number
- Implement debounced input validation for StakingTime
- Remove PreviewModal and integrate signing process into main dialog
- Adjust input types and error handling
- Improve form reset and dialog closure behavior

desamtralized committed:
better handling of stake btn while 'loading'

desamtralized committed:
refined staking modal layout

desamtralized committed:
reponsive staking dialog

desamtralized committed:
created general dialog with new layout

desamtralized committed:
general error handling through standard dialog

desamtralized committed:
moving repeated css values to tailwind config

desamtralized committed:
fix: build errors
